src/pages/create.tsx

```diff
@@ -0,0 +1,183 @@
+import { useState } from "react";
+import { Text, Button, Input } from "../components/atomic";
+import {
+  useCreateProgramMutation,
+  CreateProgramInput,
+} from "../generated/graphql";
+
+const BlobCircle = () => {
+  const sizes = "h-24 w-24 md:h-64 md:w-64 lg:h-80 lg:w-80";
+  return (
+    <div
+      className={`${sizes} rounded-full bg-primary flex justify-center items-end pointer-events-none`}
+    >
+      <img src="/static/CreateBlobs.svg" className="w-11/12 select-none" />
+    </div>
+  );
+};
+
+const CreateProgramPage = () => {
+  const [stage, setStage] = useState(0);
+  const [programName, setProgramName] = useState("");
+  const [programLogo, setProgramLogo] = useState("");
```

Comment:
May need to talk to design about the default for a mentorship's logo so that when we try to render, it doesn't try to display an empty string

Reply:
yesss will do once i uh figure out file inputs. maybe another CR tho?